README.md

```diff
@@ -2,145 +2,174 @@
+
+We strive to keep things as simple and standard as possible and base our work off the popular guide: [Node.js Best Practices](https://github.com/goldbergyoni/nodebestpractices)
 
 <br/>
 
-# `Table of contents`
-
-- [`A very quick start`](#a-very-quick-start)
-- [`Our philosophy and unique value`](#our-philosophy-and-unique-value)
-- [`Example applications (place holder, not ready yet)`]()
-  - [Express, Postgresql, with common best practices](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
-  - [Express, Mongodb, with common best practices](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
-  - [Express, Postgresql, with all best practices (advanced)](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
-  - [Minimal with project setup configuration only](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
-  <details><summary>More flavours</summary>
-  
-  - [soon] fastify, postgresql
-  
-  - [soon] fastify, mongo
-  
-  - [Generate your own interactively]()
-  
-  - More coming soon
-  </details>
-
-- [`Practices and features`](#practices-and-features)
-- [`Documentation site (coming soon)`](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
-- [`YouTube channel (soon)`](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
-- [`The people behind this project`](#the-people-behind-this-project)
+# `Table of Contents`
+
+- [`Installation and Usage`](#installation-and-usage)
+- [`Our Philosophies and Unique Values`](#our-philosophies-and-unique-values)
+- [`Practices and Features`](#practices-and-features)
+- [`The People Behind Practica.js`](#the-people-behind-practicajs)
 - [`Our best practices guide, 77,000 stars ✨`](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
 - [`Contribution guide`](https://github.com/practicajs/practica/blob/main/CONTRIBUTING.md)
+- Coming Soon:
+	- [`Documentation`](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
+	- [`YouTube`](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
+	- Example Applications
+		- [Express, PostgreSQL, with common best practices](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
+		- [Express, mongo-db, with common best practices](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
+		- [Express, PostgreSQL, with all best practices (advanced)](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
+		- [Minimal with project setup configuration only](https://github.com/practicajs/practica/blob/main/docs/not-ready-yet.md)
+		<details><summary>More Flavours</summary>
+		
+		- Fastify, PostgreSQL
+		- Fastify, mongo-db
+		- Generate Your Own Interactively
+		- More coming soon
+		</details>
+
+<br />
+
+# Installation and Usage
```

Comment:
Highly prefer punchy/bold language here, let's use "A very quick guide" or something that stands out. People tend to leave early unless they quickly find what they need